mythtv/libs/libmythbase/compat.h

```diff
@@ -137,13 +140,17 @@
 //used in videodevice only - that code is not windows-compatible anyway
 #    define minor(X) 0
 
-    using uint = unsigned int;
+	#if defined(__cplusplus)
+            using uint = unsigned int;
+        #else
+            typedef unsigned int uint;
+   #endif
```

Comment:
This change shouldn't be necessary.

Reply:
I get an error "unknown type name 'using';" when removing it. I note dvdnav.h from the external folder includes compat.h. 
In file included from dvdnav/dvdnav/dvdnav.h:42,
                 from dvdnav/read_cache.c:36:
Mythtv 0.27 use to use "typedef unsigned int uint;" and it has been changed at some point. 
I can change the "if define" from "cplusplus" to if "ms visual c" if this is preferred?

Reply:
Sorry, I didn't remember that the dvdnav code included this file. I withdraw my comment. Your change is fine.